diffpy/pdfmorph/pdfmorphapp.py

```diff
@@ -224,12 +271,11 @@ def custom_error(self, msg):
     return parser
 
 
-def main():
-    parser = create_option_parser()
-    (opts, pargs) = parser.parse_args()
-
-    if len(pargs) != 2:
-        parser.error("You must supply FILE1 and FILE2")
+def single_morph(parser, opts, pargs, stdout_flag=True):
+    if len(pargs) < 2:
+        parser.error("You must supply FILE1 and FILE2.")
+    elif len(pargs) > 2:
+        parser.error("Too many arguments.")
```

Comment:
with users in mind, when things fail it is always more helpful, as well as telling them why it failed, telling them what they should do next to make it work properly.  Much less frustrating for the user.